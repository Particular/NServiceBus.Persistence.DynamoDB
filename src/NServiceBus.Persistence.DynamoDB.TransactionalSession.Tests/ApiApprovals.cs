namespace NServiceBus.Persistence.DynamoDB.TransactionalSession.Tests
{
    using NServiceBus.TransactionalSession;
    using NUnit.Framework;
    using Particular.Approvals;
    using PublicApiGenerator;

    [TestFixture]
    public class ApiApprovals
    {
        [Test]
        public void Approve()
        {
            var publicApi = typeof(DynamoOpenSessionOptions).Assembly.GeneratePublicApi(new ApiGeneratorOptions
            {
                ExcludeAttributes = new[] { "System.Runtime.Versioning.TargetFrameworkAttribute", "System.Reflection.AssemblyMetadataAttribute" }
            });
            Approver.Verify(publicApi);
        }
    }
}