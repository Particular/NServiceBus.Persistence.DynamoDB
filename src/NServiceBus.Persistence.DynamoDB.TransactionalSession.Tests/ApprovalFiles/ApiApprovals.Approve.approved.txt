namespace NServiceBus.TransactionalSession
{
    public class DynamoOpenSessionOptions : NServiceBus.TransactionalSession.OpenSessionOptions
    {
        public DynamoOpenSessionOptions() { }
    }
    public static class DynamoTransactionalSessionExtensions
    {
        public static NServiceBus.PersistenceExtensions<NServiceBus.DynamoDBPersistence> EnableTransactionalSession(this NServiceBus.PersistenceExtensions<NServiceBus.DynamoDBPersistence> persistenceExtensions) { }
    }
}